script.js

```diff
@@ -1,31 +1,109 @@
+    liveVersion: 'https://ehmaddd.github.io',
+    linkSource: 'https://github.com/ehmaddd/ehmaddd.github.io',
+  },
+  {
+    image: './images/proj-6.jpg',
+    title: 'Multi-Post Stories Gain + Glory',
+    languages: ['Ruby', 'HTML', 'CSS', 'PHP'],
+    details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release.",
+    liveVersion: 'https://ehmaddd.github.io',
+    linkSource: 'https://github.com/ehmaddd/ehmaddd.github.io',
+  },
+];
+
+let worksUl;
+const modalUl = document.querySelector('#myModal ul');
+
+// Populating Portfolio
+
+const works = document.querySelectorAll('.work-sample');
+for (let a = 0; a < works.length; a += 1) {
+  document.querySelectorAll('.work-sample img')[a].src = projects[a].image;
+  document.querySelectorAll('.work-sample h4')[a].innerHTML = projects[a].title;
+  worksUl = document.querySelectorAll('.work-sample ul');
+
+  for (let b = 0; b < projects[a].languages.length; b += 1) {
+    const li = document.createElement('li');
+    li.appendChild(document.createTextNode(projects[a].languages[b]));
+    li.setAttribute('id', 'element4');
+    worksUl[a].appendChild(li);
+  }
+  // document.querySelectorAll('.work-sample a')[a].href = "project-"+parseInt(a+1);
+}
+
+// Populting Modal
+const modal = document.getElementById('myModal');
+const btn = document.getElementsByClassName('see-project-btn');
+const span = document.getElementsByClassName('close')[0];
+
+for (let a = 0; a < btn.length; a += 1) {
+  btn[a].addEventListener('click', () => {
+    modal.style.display = 'block';
+
+    document.querySelector('#myModal img').src = projects[a].image;
+    document.querySelector('#myModal p').innerHTML = projects[a].details;
+    document.querySelector('#myModal h4').innerHTML = projects[a].title;
+
+    while (modalUl.firstChild) {
+      modalUl.removeChild(modalUl.firstChild);
     }
+
+    for (let b = 0; b < projects[a].languages.length; b += 1) {
+      const li = document.createElement('li');
+      li.appendChild(document.createTextNode(projects[a].languages[b]));
+      // li.setAttribute("id", "element4");
+      modalUl.appendChild(li);
+    }
+
+    document.querySelector('#see-live').href = projects[a].liveVersion;
+    document.querySelector('#see-source').href = projects[a].linkSource;
```

Comment:
- [ ]  When I hover over these buttons unexpected effects happen as in the picture below so please make sure to add the effects on the Figma in order to match the Figma design 
- Yours 
<img width="178" alt="Screenshot 2023-03-08 at 13 01 48" src="https://user-images.githubusercontent.com/61790587/223709275-5a5b4fd8-2e8c-4f3b-97c0-eb0a735e2daf.png">
- Figma 
<img width="523" alt="Screenshot 2023-03-08 at 13 07 06" src="https://user-images.githubusercontent.com/61790587/223709374-6593bfb8-2ea4-4012-bd0d-0a7c377c11cc.png">

Reply:
According to the instructions and color, hover, pressed guideline by figma template the code is corrected as desired.
Regards